### PR TITLE
feat(QE): QE-203 Add Tests for Common Page Objects and Helpers

### PR DESCRIPTION
**Pull request by MarkJCram**
Jira: [QE-203](https://jira.bigcommerce.com/browse/QE-203)

## What? Why?
Common page objects and helpers within the framework that will remain and be distributed need to be validated before distribution.  Basic functional/sanity tests need to be written to cover the framework and alert the team to any necessary updates before the framework is distributed and potentially breaks downstream tests.

### CODE REVIEW

File: src/helpers/cp-side-nav-helper.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
+const { I, cpSideNavPage } = inject();
```
Comment (BC-Ubalderas) [inline src/helpers/cp-side-nav-helper.js:1]: Similarly to the user settings helper, I think these methods could live in the cpSideNavPage instead, but I think we should discuss and agree to a pattern.

**Review by BC-Ubalderas — COMMENTED** [review]
Left a couple of comments, we should have a follow up discussion before approval.

File: src/helpers/user-settings-helper.js [hunk 2]
```diff
@@ -1,38 +1,45 @@
-import { selectors } from '../pages/user-settings-page';
-import { DEFAULT_TIMEOUT } from '../global-constants';
-
-module.exports = function() {
-
-    return actor({
+const { I, userSettingsPage } = inject();
```
[the remark this answers is not in the document]
Comment (MarkJCram) [reply src/helpers/user-settings-helper.js:1]: I definitely agree and plan to follow up with this in a subsequent PR

**Review by BC-Ubalderas — APPROVED** [review]
Per our conversation, re-structuring of the helpers methods and page methods will happen on a follow up PR, and the scope for this one was just for consistent patterns across our helpers and page methods.

**Pull request merged**